### DISCUSSION TITLE
Maybe credit PureRef by name somewhere

### DISCUSSION THREAD

**Discussion by sidney-eliot** [rbreu/beeref#72] [Ideas & General Discussion]
There's nothing bad about re-creating PureRef, but maybe credit it somewhere in the readme as the inspiration for BeeRef. 

I'm personally quite happy that this "fork" of PureRef exists. They sadly abandoned it and never made it open source :(

**Comment by pranabekka** [comment]
There is also a selfish reasons to this: people searching for PureRef are more likely to stumble across BeeRef.

Also, anyone who knows and/or likes PureRef might not be comfortable with BeeRef if it doesn't at least acknowledge the similarity.

The mention could also include why BeeRef exists: PureRef was abandoned, and this version is free/open-source forever — available for the entire community to fix, enhance and carry on into the future.

**Reply by sidney-eliot** [reply]
Little heads up, BeeRef is just as abandoned as PureRef at this point and it’s been abandoned for quite a long time now. Don’t even bother using this, just stick to PureRef.

The only solution I see in getting an actively developed ref board software is if someone like adobe steps in and makes one, but I don’t think that will ever happen and I also don’t think we would want some big corpo trying to commercialize that.

A possibility is Blender might replace PureRef, they are working on making Blender more module to allow people to create new software based on Blender like a 3d printer software, made of blender which only contains that which is needed.

Lastly if you want a cloud version of PureRef use https://milanote.com/, it has so many cool features

**Reply by pranabekka** [reply]
> Little heads up, BeeRef is just as abandoned as PureRef at this point and it’s been abandoned for quite a long time now. Don’t even bother using this, just stick to PureRef.

:p True. I realised later as I continued going through the project

However, [mini-ninja-64](https://github.com/mini-ninja-64/beeref) seems to be steadily chipping away at it.

**Comment by cheshirecatt** [comment]
I don't think it's necessary to do that, after all this isn't a fork without mentioning that PureRef isn't the only app to be a reference board, there's the also discontinued Kuadro.

**Comment by sidney-eliot** [comment]
Ok fair point PureRef isn't the first, Kuadro already existed in 2015. I retract my opinion that BeeRef should credit anyone as there is no way of saying who started it and quite honestly it doesn't really matter, thanks for setting me straight @cheshirecatt
